src/redux/userReducer.js

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+const TOKEN =
+  "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYxMjNmMTdlNWMzY2IxNDVjMDQ1ZTk0ZSIsInJvbGUiOiJhZG1pbiIsImlhdCI6MTYzMDk1NTk0NCwiZXhwIjoxNjMxODM0MzQ0fQ.AXc0KhpkkR9d475zpfQVH1nXugZ1SJomyQ1uJBSerRw";
+
+export const userFetch = createAsyncThunk(
+  "user/fetch",
+  async function (_, { rejectWithValue }) {
+    try {
+      const response = await fetch("http://localhost:3000/users/me", {
```

Comment:
it's better to get `http://localhost:3000` from a constant variable, because this url is going to be changed

Reply:
you'd better have general function called `formatRequest` (anything like that) which adds all general headers (as `Authorization`), parses response from JSON, gets the value from response and handles errors and something like that